.circleci/config.yml

```diff
@@ -4,10 +4,12 @@ aliases:
   - &restore-node-modules-cache
     keys:
       - v1-yarn-deps-{{ checksum "yarn.lock" }}
+      - v1-yarn-deps-
```

Comment:
Why do we need this key without the hash?

Reply:
Good question!

It is intended to fetch the latest cache generated successfully by any branch if the `yarn.lock` is changed. Chances are that there could be some cache contents that can still be reused by `yarn`, i.e. some libraries that are not still resolved by the changed lock file.

See also https://circleci.com/docs/2.0/caching/#restoring-cache